src/Core/Resolvers/DWSqlQueryBuilder.cs

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Data.Common;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Azure.DataApiBuilder.Core.Models;
+using Microsoft.Data.SqlClient;
+
+namespace Azure.DataApiBuilder.Core.Resolvers
+{
+    /// <summary>
+    /// Class for building DwSql queries.
+    /// </summary>
+    public class DwSqlQueryBuilder : BaseSqlQueryBuilder, IQueryBuilder
+    {
+        private static DbCommandBuilder _builder = new SqlCommandBuilder();
+
+        /// <inheritdoc />
+        public override string QuoteIdentifier(string ident)
+        {
+            return _builder.QuoteIdentifier(ident);
+        }
+
+        /// <summary>
+        /// Builds the sql query that will return the json result for DW.
+        /// Sample: Get title of all books.
+        /// SELECT COALESCE('['+STRING_AGG('{"title":"' + STRING_ESCAPE(ISNULL(title,''),'json') + '"}',', ')+']','[]')
+        /// FROM (
+        ///     SELECT TOP 100 [table0].[title] AS [title]
+        ///     FROM [dbo].[books] AS [table0]
+        ///     WHERE 1 = 1 ORDER BY [table0].[id] ASC
+        ///     ) AS [table0]
+        /// </summary>
+        public string Build(SqlQueryStructure structure)
+        {
+            string columns;
```

Comment:
Can you note the significance of one set to string.Empty and the other using the default of null?

Reply:
Fixed after review in dab sync up